source/ui/Components/Output/LightstripSettingsWindow.qml

```diff
@@ -106,8 +115,12 @@ Window {
                 // Reads the current input and passes the new value to the model.
                 function processInput() {
                     var inputValue = controlsColumn.integerFromInputString(String(numberOfStripsTextField.text));
-                    if (inputValue !== undefined && inputValue > 0) {
+                    var size = outputComponent.getMemorySize();
+                    var maxStrips = (outputComponent.getMemorySize() - size) * 8;
+                    if (inputValue !== undefined && inputValue > 0 && inputValue <= maxStrips) {
```

Comment:
I would suggest to check for too many digits separately and default to the highest possible value (maxStrips) in this case. I think that's more intuitive for the user.